src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -226,7 +224,6 @@ public void SarifTransformerTests_ToCurrent_OneRunWithLogicalLocations()
           ""kind"": ""type""
         },
         ""collections"": {
-          ""name"": ""collections"",
```

Comment:
Why did this test not fail before your last merge?

Reply:
That change affected the converters, where this one is the transformer. I thought of something we need to check though, which is that the logger is doing this correctly.